extensions/amp-ad/0.1/amp-ad-custom.js

```diff
@@ -80,6 +80,12 @@ export class AmpAdCustom extends AMP.BaseElement {
       'custom ad slot should be alphanumeric: ' + this.slot_
     );
 
+    const urlService = Services.urlForDoc(this.element);
+    userAssert(
```

Comment:
QQ: Is the check necessary if we already have
```
value_url: {
      protocol: "https"
    }
```
in validator check

Reply:
User asserts are for local development, too.